git/week1/jsweek1.js

```diff
@@ -0,0 +1,86 @@
+// Homework javascript1-week1
+
+//  Age-ify (A future age calculator)
+
+var yearOfBirth = 1987;
+var yearFuture = 2030;
+var age = yearFuture - yearOfBirth;
+console.log('You will be ' + age + ' years old in ' + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+
+var dogYearOfBirth = 2020
+var dogYearFuture = 2030;
+var yearDifference = dogYearFuture - dogYearOfBirth;
+var dogYear = 7 * (yearDifference);
+let shouldShowResultInDogYears = 'false';
+
+if (shouldShowResultInDogYears == 'true') {
+    console.log('Your dog will be ' + dogYear + ' dog years in ' + dogYearFuture)
+}
+else {
+    console.log('Your dog will be ' + yearDifference + ' Human years in ' + dogYearFuture)
+}
+
+// Housey pricey (A house price estimator)
+// evaluation for Peter 
+
+var wideForPeter = 8;
```

Comment:
Imagine you can put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8]; ` `const gardenSizeInM2 = [100,70];`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?